utils/nostr.ts

```diff
@@ -376,46 +380,93 @@ export const getWriteRelayUris = (event: Event) => {
     .map((tag) => tag[1]);
 };
 
-export const fetchInvoice = async ({
-  relayUris,
+export const makeZapRequest = async ({
+  contentId,
+  parentContentType,
   amountInSats,
+  relays = [],
   comment,
-  addressPointer,
-  zappedPubkey,
   timestamp,
-  zapEndpoint = "https://www.wavlake.com/api/zap",
   customTags = [],
 }: {
-  relayUris: string[];
+  contentId: string;
+  parentContentType: "podcast" | "album" | "artist";
   amountInSats: number;
+  relays?: string[];
   comment: string;
-  addressPointer: string;
-  zappedPubkey: string;
   timestamp?: number;
-  zapEndpoint?: string;
   customTags?: EventTemplate["tags"];
-}): Promise<{ pr: string } | { status: string; reason: string }> => {
-  const wavlakeRelayUri = "wss://relay.wavlake.com/";
-  const amountInMillisats = amountInSats * 1000;
+}): Promise<EventTemplate> => {
+  const nostrEventAddressPointer = `${wavlakeTrackKind}:${wavlakePubkey}:${contentId}`;
+  const iTags = [
+    ["i", `podcast:item:guid:${contentId}`],
+    ["i", `podcast:guid:${getPodcastFeedGuid(parentContentType, contentId)}`],
+    [
+      "i",
+      `podcast:publisher:guid:${getPodcastFeedGuid(
+        parentContentType,
+        contentId,
+      )}`,
+    ],
+  ];
   const zapRequestEvent = await nip57.makeZapRequest({
-    profile: zappedPubkey,
-    amount: amountInMillisats,
-    relays: [...relayUris, wavlakeRelayUri],
+    profile: wavlakePubkey,
```

Comment:
Should we use the new env var / variable naming convention here?
```suggestion
    profile: walletServicePubkey,
```